xstatic/pkg/font_awesome/__init__.py

```diff
@@ -46,9 +49,5 @@
     # your relative path/filenames should usually be without version
     # information, because either the base dir/url is exactly for this
     # version or the mapping will care for accessing this version.
-    ('cdnjs', 'http'): 'http://cdnjs.cloudflare.com/ajax/libs/font-awesome/%s/' % VERSION,
```

Comment:
the http url works in the same way as the https url in the next line.

guess it is not our decision, but the decision of whoever uses the CDN URLs. so we should also provide the http URL.